start-minecraftFinalSetup

```diff
@@ -205,6 +205,14 @@ EOF
     sed -i "s/MAX_RAM=[^;]*/MAX_RAM=${MAX_MEMORY}/" "${FTB_DIR}/settings.cfg"
   fi
 
+  # patch CurseForge settings.sh file, if present
+  if [ -f "${FTB_DIR}/settings.sh" ]; then
+    sed -i "s/MIN_RAM=[^;]*/MIN_RAM=\"${INIT_MEMORY}\"/" "${FTB_DIR}/settings.sh"
+    sed -i "s/MAX_RAM=[^;]*/MAX_RAM=\"${MAX_MEMORY}\"/" "${FTB_DIR}/settings.sh"
```

Comment:
The replacement string might need to retain the trailing semicolon, since I seem to remember the start scripts blindly stripping the last character.

Reply:
So you'd want something like `MIN_RAM=.*[;]?` instead?

Reply:
No, rather I'm wondering if it needs to

```suggestion
    sed -i "s/MIN_RAM=[^;]*/MIN_RAM=\"${INIT_MEMORY}\";/" "${FTB_DIR}/settings.sh"
    sed -i "s/MAX_RAM=[^;]*/MAX_RAM=\"${MAX_MEMORY}\";/" "${FTB_DIR}/settings.sh"
```

however, if you've tested and it works leave it as is.

Reply:
> however, if you've tested and it works leave it as is.

My server has been running with these changes for 2 days. It is working exactly as expected in my very specific case.